packages/yourloops/models/shoreline.ts

```diff
@@ -96,7 +96,8 @@ interface IUser {
   preferences?: Preferences | null;
   /** Patient medical data. undefined means not fetched, null if the fetch failed */
   medicalData?: MedicalData | null;
-  alerts?: Alarm | null;
+  alarms?: Alarm | null;
+  monitoring?: { enabled: boolean}
```

Comment:
| null ?

Reply:
I don't know anymore, if it is ok with you I'd like to keep it as is